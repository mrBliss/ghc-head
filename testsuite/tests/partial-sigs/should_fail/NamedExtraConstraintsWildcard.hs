{-# LANGUAGE PartialTypeSignatures, NamedWildcards #-}
module NamedExtraConstraintsWildcard where

foo :: (Eq a, _a) => a -> a
foo = undefined
