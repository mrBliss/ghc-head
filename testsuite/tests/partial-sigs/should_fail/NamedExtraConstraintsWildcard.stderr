
NamedExtraConstraintsWildcard.hs:4:8:
    Invalid partial type signature: (Eq a, _a) => a -> a
    A named wildcard cannot occur as a constraint
